data/mp/multiplay/skirmish/bonecrusher/functions.js

```diff
@@ -433,7 +433,7 @@ function playerLoose(player){
 
 function playerSpectator(player){
 	var loose = false;
-	if( ( enumStruct(player, "A0Sat-linkCentre").length === 1 || enumStruct(player, "A0CommandCentre").length === 1 )
+	if (( enumStruct(player, "A0Sat-linkCentre").length === 1 || enumStruct(player, "A0CommandCentre").length === 1)
```

Comment:
One more :)

Could remove the space before that `enumStruct()` on the leftmost side now.